release-notes/topics/controller-412.adoc

```diff
@@ -0,0 +1,19 @@
+[[controller-412-intro]]
+= Automation controller 4.1.2
```

Comment:
Automation Controller 4.2.0 will be shipped as part of AAP 2.2 (not 4.1.2 as wrttten here)

Reply:
Fixed.